client/src/components/login/RegisterForm.vue

```diff
@@ -54,10 +78,14 @@ import Vue from "vue";
 import BootstrapVue from "bootstrap-vue";
 import { getGalaxyInstance } from "app";
 import { getAppRoot } from "onload";
+import ExternalLogin from "components/User/ExternalIdentities/ExternalLogin.vue";
```

Comment:
No need to end it with `.vue` here.

Reply:
So actually it works without due to webpack magic, but there's no need to rely on this.  This is one of those explicit is better than implicit times, and it doesn't buy us anything significant to drop the extension.  This is another one of those where I wish we'd been explicit the whole time, and things would be in a more consistent state.

Reply:
I agree, explicit is usually better and does not hurt but it's not what we seem to have used throughout the code base in the recent years. For consistency it would be nice if we agree on how we do these imports and for now it seems its to do it implicit.

Reply:
Yep, unfortunate that the shift crept in.  I'll put it on the agenda for the next UI/UX meeting and we can pick a stance there and stick with it.

Reply:
In my opinion, we should keep it implicit 1) It avoids duplicate filenames within the same directory, 2) its what we have mostly used in the recent years and 3) its less to write.

Reply:
So, I can go ahead and keep it explicit. Other than that, all good with the PR?
The only API test that failed had something to do with workflows:
![image](https://user-images.githubusercontent.com/78516064/166503743-b205cb0c-7589-40ca-bfac-773f0bda32bd.png)
... which is not related to the changes I have made here.

Reply:
@ahmedhamidawan Yep; this part is fine.  I'll retest the rest of the PR today.